wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -449,6 +452,13 @@ public void run() {
   @Traced
   public CompletableFuture<Void> stopAsync(Workspace workspace, Map<String, String> options)
       throws NotFoundException, ConflictException {
+    TracingTags.WORKSPACE_ID.set(workspace.getId());
+    TracingTags.STOPPED_BY.set(
+        () ->
+            firstNonNull(
```

Comment:
This code looks overweight. Maybe we can leave SubjectID as is? @sleshchenko @mshaposhnik @metlos wdyt?

Reply:
I think that `stopped by` tag could be really useful in traces. Well, maybe you would like code with private method and method reference?
```
public CompletableFuture<Void> stopAsync(Workspace workspace, Map<String, String> options) throws NotFoundException, ConflictException {
    TracingTags.WORKSPACE_ID.set(workspace.getId());
    TracingTags.STOPPED_BY.set(() -> getStoppedBy(workspace));
     ...
}
```

Reply:
done like suggested